.github/workflows/generate_full_dist.yml

```diff
@@ -14,16 +14,17 @@ permissions:
   contents: write
   pull-requests: write
 
-env:
-  GITHUB_SHA_SHORT: $(git rev-parse --short ${{ github.sha }})
-
 jobs:
   generate_full_dist:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout code
       uses: actions/checkout@v4
 
+    - name: Set up short SHA
+      id: short_sha
+      run: echo "::set-output name=sha::$(git rev-parse --short ${{ github.sha }})"
```

Comment:
Maybe works betetr as a job env? Also I think there's a new format for setting step outpuyt (this format is deprecated)